site/cypress/e2e/Cypress-UI/pronouns_change.spec.js

```diff
@@ -58,7 +55,7 @@ describe('Tests cases abut changing user pronouns', () => {
         cy.get('#toggle-student-col-submit').first().click();
 
         //Ensure correctness in table
-        cy.get('.td-pronouns:eq( 12 )').should('have.text', 'They/Them');
+        cy.get('.td-pronouns:eq( 12 )').should('have.text', newPronouns);
```

Comment:
Ideally this would use a different selector, one that wouldn't break if the order of the students changes.

Reply:
Yea, I definitely agree. But I would argue that isn't the concern for the issue that this PR is trying to address.

Reply:
I agree, hence not a requested change, just a comment.